deepmd/entrypoints/transfer.py

```diff
@@ -89,6 +96,38 @@ def load_graph(graph_name: str) -> tf.Graph:
         tf.import_graph_def(graph_def, name="")
         return graph
 
+def modify_const_op(new_graph_def: tf.Graph) -> tf.Graph:
+    """modify natoms to constant.
+
+    Parameters
+    ----------
+    new_graph : tf.Graph 
+        orginal new graph
+    Returns :
+    -------
+    tf.Graph
+        natoms transfer to a const op for Ascend platform
+    -------
```

Comment:
The correct format:
```suggestion

    Returns
    -------
    tf.Graph
        natoms transfer to a const op for Ascend platform
```

Reply:
Thanks, we resolved the conflicts and fixed the reference.